travis-ci/managers/xenial.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+set -e
+set -x
+
+add-apt-repository ppa:pitti/systemd-semaphore -y
+apt-get update
+apt-get build-dep systemd -y
+apt-get install -y util-linux libmount-dev libblkid-dev liblzma-dev libqrencode-dev libmicrohttpd-dev iptables-dev liblz4-dev libcurl4-gnutls-dev unifont itstool kbd cryptsetup-bin net-tools isc-dhcp-client iputils-ping strace qemu-system-x86 linux-image-virtual mount libgpg-error-dev libxkbcommon-dev python-lxml python3-lxml python3-pip libcap-dev
+apt-get install -y gettext python3-evdev python3-pyparsing libmount-dev python3-setuptools ninja-build
+pip3 install meson
+
+cd $REPO_ROOT
+
+sed -i 's/2\.30/2.27/' meson.build
```

Comment:
@martinpitt could you upload `libmount-dev-2.30.1` to http://ppa.launchpad.net/pitti/systemd-semaphore/ubuntu/dists/xenial so that I can remove this kludge?

Reply:
I suppose this is obsolete since moving CI to ubuntu 18.04 (bionic)?

Reply:
This script is run on Travis CI where Ubuntu Xenial is the latest version of Ubuntu available: https://docs.travis-ci.com/user/reference/overview/#linux. I switched to it to get around overlayfs bugs like the one fixed in https://github.com/systemd/systemd/pull/11095/commits/f413cd0a6af10c08561474a089f8670979c0f786 and to get `libasan` that wouldn't be too old.

Reply:
[As it turns out](https://github.com/systemd/systemd/pull/11531#issuecomment-457305488), Ubuntu Xenial is also used on SemaphoreCI, where we didn't have any problems with `libmount-dev` because it was installed from the "artful/main" repository.

Reply:
This is in progress..... Will report once all the builds done.

Reply:
I think it's published now. Can you retry any semaphore thing? https://launchpad.net/~upstream-systemd-ci/+archive/ubuntu/systemd-ci/+packages?field.name_filter=&field.status_filter=published&field.series_filter=xenial

Reply:
Great! Thank you! Unfortunately, I don't have access to semaphoreci. I think it will be triggered if you force-push https://github.com/systemd/systemd/pull/11531.

Reply:
I triggered it via the webhook: https://semaphoreci.com/systemd/systemd/branches/pull-request-8608/builds/5.

Reply:
I triggered it once more just in case: https://semaphoreci.com/systemd/systemd/branches/pull-request-8608/builds/6. Apparently something is wrong with the kbd package:
```
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 kbd : Depends: console-setup but it is not going to be installed or
                console-setup-mini but it is not going to be installed
E: Unable to correct problems, you have held broken packages.
```

Reply:
In https://github.com/systemd/systemd/pull/11582 I'm in the process of switching to the new repository and removing the kludge. @martinpitt @xnox thanks!